zuul.d/edpm_build_images.yaml

```diff
@@ -26,3 +26,18 @@
       - ^ci/playbooks/edpm_build_images/.*
       - ^scenarios/centos-9/edpm_build_images_ci.yml
       - ^zuul.d/edpm_build_images.yaml
+
+- job:
+    name: periodic-edpm-build-push-images-centos-9-antelope
+    parent: cifmw-base-edpm-build-images
+    vars:
+      promote_source: podified-ci-testing
+      push_registry: quay.rdoproject.org
+      registry_namespace: podified-antelope-centos9
+      push_containers: true
+      buildah_login: true
+      registry_login_enabled: true
+      openstack_release: antelope
+      cifmw_repo_setup_branch: antelope
+      trunk_url: https://trunk.rdoproject.org/
```

Comment:
@bshewale where do we use trunk_url?

Reply:
hey those where just copy pasted, we can clean up those stuff once we push the images successfully.

Reply:
ack, thanks for the update. removed in the latest patchset, let's see how it goes.

Reply:
looks like We don't need this, removed